user/cinnamon-settings-daemon/patches/XyG2.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/install-scripts/meson_mk_symlink.py b/install-scripts/meson_mk_symlink.py
```

Comment:
this patch shouldn't be necessary since c9d75988b64479395b86ebacafbda031c8aedd8c (though the presence of the script at all isn't ideal, but i'll address that upstream)